src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/pact/provider/ManageRecallsUIPactTest.kt

```diff
@@ -84,10 +84,10 @@ class ManagerRecallsUiAuthorizedPactTest : ManagerRecallsUiPactTestBase() {
 
   @State("a prisoner exists for NOMS number")
   fun `a prisoner exists for NOMS number`() {
-    mockPrisonerResponse(nomsNumber)
+    mockPrisonerResponses(nomsNumber)
```

Comment:
So we're not adding to the contract for the return of a single prisoner outside of a List/Array?